rust/dyn-ddlog/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "dyn-ddlog"
+version = "0.1.0"
+authors = ["Chase Wilson <me@chasewilson.dev>"]
+edition = "2018"
+build = "build.rs"
+default-run = "ddlog"
+
+[features]
+default = ["debug-printing"]
+debug-printing = ["pretty"]
+
+[[bin]]
+name = "ddlog"
+required-features = ["structopt"]
+
+[dependencies]
+egg = "0.6.0"
+regex = "1.4.5"
+lalrpop-util = "0.19.5"
+pretty = { version = "0.10.0", optional = true }
+structopt = { version = "0.3.21", optional = true }
+
+[dev-dependencies]
+goldentests = "0.3.7"
```

Comment:
One thing you may want to do early on is start running whatever compilation steps you currently support on the complete DDlog test suite. I'm not suggesting you reuse the current test framework based on scripts, but just the actual `.dl` files (and eventually `.dat`s, and `.expected`).

Cons:
- requires a bunch of extra work upfront, in particular support for full DDlog syntax.

Pros:
- better test coverage 
- enforces compatibility with the Haskell compiler.

Reply:
Yes, that's definitely something I want to do soon